compiler/src/iree/compiler/Codegen/ExternalInterfaces/Utils.h

```diff
@@ -7,11 +7,19 @@
 #ifndef IREE_COMPILER_CODEGEN_EXTERNALINTERFACES_UTILS_H_
 #define IREE_COMPILER_CODEGEN_EXTERNALINTERFACES_UTILS_H_
 
+#include "mlir/IR/Builders.h"
 #include "mlir/IR/BuiltinAttributes.h"
 #include "mlir/IR/BuiltinTypes.h"
+#include "mlir/IR/ValueRange.h"
 
 namespace mlir::iree_compiler::IREE {
 
+static StringLiteral kEncodingInfoAttrName = "encoding_info";
```

Comment:
`static const char kEncodingInfoAttrName[]` to avoid adding a relocation when the `StringLiteral` constructor takes the address of the literal. https://github.com/iree-org/iree/issues/19529